install/kubernetes/cilium/files/nodeinit/prestop.bash

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
I am not sure if this change makes it's easier to read bash script, as this file is now mixed with go template and bash.

I checked this one with below commands and compared the diff, seems like there are some changes with indentation (e.g. tab, space, etc), it might not be issue, but no harm to have second :eyes: 

```
# From this branch
helm template ./cilium --set nodeinit.enabled=true --set nodeinit.revertReconfigureKubelet=true > new.yaml

# From the master
helm template ./cilium --set nodeinit.enabled=true --set nodeinit.revertReconfigureKubelet=true > original.yaml

diff original.yaml new.yaml
```
```

Reply:
Thank @sayboras , I add `-` to gotpl open delimiter to make them more identical. You could 
```
{{- tpl (.Files.Get "files/nodeinit/prestop.bash") . | nindent 20 }}
```

And yeah, I know that script is now mixed with go template and bash, but with code highlight from IDE, I think it's much easier to read than previous:
Old:
![Screenshot from 2021-07-07 22-18-27](https://user-images.githubusercontent.com/6848311/124786840-4281ca00-df72-11eb-8ea2-2a7df7cfef79.png)
New:
![Screenshot from 2021-07-07 22-17-30](https://user-images.githubusercontent.com/6848311/124786836-40b80680-df72-11eb-8608-6591fd00783c.png)

Reply:
Thank @sayboras , I fixed the spaces in nodeinit scripts

And yeah, I know that script is now mixed with go template and bash, but with code highlight from IDE, I think it's much easier to read than previous:
Old:
![Screenshot from 2021-07-07 22-18-27](https://user-images.githubusercontent.com/6848311/124786840-4281ca00-df72-11eb-8ea2-2a7df7cfef79.png)
New:
![Screenshot from 2021-07-07 22-17-30](https://user-images.githubusercontent.com/6848311/124786836-40b80680-df72-11eb-8608-6591fd00783c.png)